platform/pulp/tasking/tasks.py

```diff
@@ -96,7 +96,7 @@ def _queue_reserved_task(name, inner_task_id, resource_id, inner_args, inner_kwa
     try:
         celery.tasks[name].apply_async(*inner_args, **inner_kwargs)
```

Comment:
Well the end task's parameters always live inside their own dict and tuple and never get mixed with others. For example I call this https://github.com/BrnoPCmaniak/pulp/blob/b6e579ca3a8ec7d50bd0127738d2403b5620f7ee/platform/pulp/tasking/tasks.py#L152 this:
```
some_task.apply_async_with_reservation("RPM", "dnf-2.5.0", tags=[], group_id=None, args=(1, 2), kwargs={"max_retries": 5}, max_retries=10)
```
this leads to https://github.com/BrnoPCmaniak/pulp/blob/b6e579ca3a8ec7d50bd0127738d2403b5620f7ee/platform/pulp/tasking/tasks.py#L45
```
_queue_reserved_task.apply_async(args=["some_task", "123e4567-e89b-12d3-a456-426655440000", "dnf-2.5.0", tuple(), {"args": (1, 2),"kwargs": {"max_retries": 5}, "max_retries": 10}], queue=RESOURCE_MANAGER_QUEUE)
```

Thanks to this method on line https://github.com/BrnoPCmaniak/pulp/blob/b6e579ca3a8ec7d50bd0127738d2403b5620f7ee/platform/pulp/tasking/tasks.py#L45 will be called taht method with parameters:

```
_queue_reserved_task("some_task", "123e4567-e89b-12d3-a456-426655440000", "dnf-2.5.0", tuple(), {"args": (1, 2),"kwargs": {"max_retries": 5}, "max_retries": 10})
```

And then the apply async is finally called.
https://github.com/BrnoPCmaniak/pulp/blob/b6e579ca3a8ec7d50bd0127738d2403b5620f7ee/platform/pulp/tasking/tasks.py#L220
```
some_task.apply_async(args=(1, 2), kwargs={"max_retries": 5}, max_retries=10)
```

Which then calls the task itself:
```
some_task(1,2 max_retries=5)
```